reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -143,7 +143,9 @@ protected PooledConnectionProvider(Builder builder) {
 			                    .stream()
 			                    .map(pool -> {
 			                        if (pool instanceof GracefulShutdownInstrumentedPool) {
-			                            return ((GracefulShutdownInstrumentedPool<T>) pool).disposeGracefully(disposeTimeout);
+			                            return ((GracefulShutdownInstrumentedPool<T>) pool)
+			                                    .disposeGracefully(disposeTimeout)
+			                                    .onErrorResume(t -> Mono.empty());
```

Comment:
you might want to just log the fact that the pool didn't shut down gracefully within the grace period, up to you.

note that acquire `Mono`s that are still _pending_ at this point will terminate with an `onError` anyway.

what doesn't get notified is already acquired `Mono` for which the pool awaits a `release()`. once the pool is shut down, that release should still happen but it will lead to a destruction of the resource.